lib/src/core.rs

```diff
@@ -19,6 +20,34 @@ pub struct Document {
     pub page_footer: Vec<Element>,
 }
 
+#[derive(Error, Debug)]
+#[cfg_attr(feature = "json", derive(Serialize, Deserialize))]
+pub enum DocumentType {
+    Html,
+    Markdown,
+    Text,
+    Pdf,
+    Json,
+    Xml,
+    Doc,
+    XLS
+}
+
+impl fmt::Display for DocumentType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            DocumentType::Html => write!(f, "Html"),
+            DocumentType::Markdown => write!(f, "Markdown"),
```

Comment:
remove unsupported format

change Html to HTML
PDF
MarkDown
JSON

Reply:
Done